zeppelin-zengine/src/main/java/org/apache/zeppelin/search/LuceneSearch.java

```diff
@@ -121,7 +125,7 @@ public LuceneSearch() {
   }
 
   private List<Map<String, String>> doSearch(IndexSearcher searcher, Query query,
-      Analyzer analyzer, Highlighter highlighter) {
+                                             Analyzer analyzer, Highlighter highlighter) {
```

Comment:
I think previous indentation was correct, according to project style, see [rationale here](https://google.github.io/styleguide/javaguide.html#s4.6.3-horizontal-alignment)